documentation/architecture/decisions/0003-use-ecs-for-initial-beta-buildout.md

```diff
@@ -0,0 +1,37 @@
+# 3. Use Amazon ECS for initial beta build-out
+
+Date: 2018-06-21
+
+(although note that this ADR was written post hoc, a couple of months
+after the decision was made)
+
+## Status
+
+Accepted.
+
+## Context
+
+Existing self-hosted infrastructure at GDS has been managed in code
+using tools like puppet, but in a somewhat ad hoc way with each team
+doing things differently, little sharing of code, and much reinvention
+of wheels.  We would like to learn about other ways of deploying
+infrastructure which encourage consistency: in terms of code
+artifacts, configuration methods, and such like.
+
+Systems such as Kubernetes and Amazon ECS are coalescing around Docker
```

Comment:
It felt like we decided to used Docker and then it narrowed our choices of architecture (although do let me know if I got the wrong impression). If so, then it would be good to say why ECS was chosen (maybe something to do with Pay using it already, Kubernetes being too excessive)